packages/components/nodes/ChainLink/supportedNetwork.ts

```diff
@@ -0,0 +1,215 @@
+import { INodeOptionsValue } from "../../src";
+import { NETWORK, NETWORK_LABEL } from "../../src/ChainNetwork";
+
+export const chainLinkNetworkMapping = {
+    [NETWORK.MAINNET]: 'Ethereum Mainnet',
+    [NETWORK.GÖRLI]: 'Goerli Testnet',
+    [NETWORK.ARBITRUM]: 'Arbitrum Mainnet',
+    [NETWORK.ARBITRUM_GOERLI]: 'Arbitrum Goerli',
+    [NETWORK.OPTIMISM]: 'Optimism Mainnet',
+    [NETWORK.OPTIMISM_GOERLI]: 'Optimism Goerli',
+    [NETWORK.MATIC]: 'Polygon Mainnet',
+    [NETWORK.MATIC_MUMBAI]: 'Mumbai Testnet',
+    [NETWORK.BSC]: 'BNB Chain Mainnet',
+    [NETWORK.BSC_TESTNET]: 'BNB Chain Mainnet',
+    [NETWORK.GNOSIS]: 'Gnosis Chain Mainnet',
+    [NETWORK.HECO]: 'HECO Mainnet',
+    [NETWORK.FANTOM]: 'Fantom Mainnet',
+    [NETWORK.FANTOM_TESTNET]: 'Fantom Testnet',
+    [NETWORK.AVALANCHE]: 'Avalanche Mainnet',
+    [NETWORK.AVALANCHE_TESTNET]: 'Avalanche Testnet',
+    [NETWORK.SOLANA]: 'Solana Mainnet',
+    [NETWORK.SOLANA_DEVNET]: 'Solana Devnet',
+    [NETWORK.HARMONY]: 'Harmony Mainnet',
+    [NETWORK.MOONRIVER]: 'Moonriver Mainnet',
+    [NETWORK.MOONBEAM]: 'Moonbeam Mainnet',
+    [NETWORK.METIS]: 'Metis Mainnet',
+    [NETWORK.KLATYN_TESTNET]: 'Klaytn Baobab testnet',
+} as any;
```

Comment:
prefer narrowly typed const assertion `as const`

if narrowed type is not compatible with other type for now, prefer type cast as index signature `as Record<string, string>`